src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.scss

```diff
@@ -0,0 +1,76 @@
+@import 'src/styles/common';
+
+.blastSubmissionResultsContainer {
+  max-width: 1860px;
+  padding: 0 $standard-gutter $global-padding-bottom 60px;
+}
+
+.sequenceBoxWrapper {
+  padding: 24px 26px; // TODO: the 26px padding looks like a constant asking to be extracted
+  border: 1px solid $grey;
+}
+
+.sequenceBoxWrapper + .sequenceBoxWrapper {
+  border-top: none;
+}
+
+.sequenceHeader {
+  width: 800px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.resultsSummaryRow {
+  display: grid;
+  grid-template-columns: 12ch 1060px auto;
+  column-gap: 20px;
+  padding-bottom: 20px;
+}
+
+.sequenceId,
+.hitLabel {
+  text-align: right;
+}
+
+.label {
+  font-weight: $light;
+  padding-right: 8px;
+}
+
+.boldText {
+  font-weight: $bold;
+  padding-right: 4px;
+}
```

Comment:
1) Check with Andrea. It doesn't look bold on her designs. Compare review deployment:

![image](https://user-images.githubusercontent.com/6834224/182549878-4bc1b541-11af-4a55-a3b7-d9429f275835.png)

with the XD:

![image](https://user-images.githubusercontent.com/6834224/182549928-5692243a-c8c7-438b-9dff-35a2fd2f4956.png)


2) Why does a class called `boldText` also have a rule about right padding?